### DISCUSSION TITLE
Honeycomb's big, yeah yeah yeah

### DISCUSSION THREAD

**Discussion by notcraig** [smartavionics/Cura#142] [General]
Wanted thank you for incorporating honeycomb as an infill option (and  the other things I'm finding you've added or worked on compared to UM version).
While I only use it for aesthetic purposes, having it in Cura means I don't have to open _that other slicer_ to do one thing.

I found your fork by way of a reddit comment when searching for a solution to not having honeycomb in Cura. The  commentor couldn't remember a link or user repo  - they referred to it as "Cura master". Imagine the challenge that presented!

Anyway, thanks again.

For anyone that is not old enough to get the reference in my title, my gift to you:
https://www.youtube.com/watch?v=RBXJ9gLlYrg

**Comment by smartavionics** [comment]
> For anyone that is not old enough to get the reference in my title, my gift to you:

I don't think that one crossed the water to the UK. I can understand how it stays in your mind forever!

**Reply by Sophist-UK** [reply]
Yeah - it stays in your mind like seeing a car crash or watching a slasher movie.

**Reply by notcraig** [reply]
To be fair, tome things happen on this side of the pond that just need to STAY on this side of the pond...
